src/plugins/discover/public/application/apps/main/components/sidebar/discover_sidebar_responsive.tsx

```diff
@@ -105,15 +105,6 @@ export interface DiscoverSidebarResponsiveProps {
    * Read from the Fields API
    */
   useNewFieldsApi?: boolean;
-  /**
-   * an object containing properties for proper handling of unmapped fields
-   */
-  unmappedFieldsConfig?: {
-    /**
-     * determines whether to display unmapped fields
-     */
-    showUnmappedFields: boolean;
-  };
```

Comment:
FYI @majagrubic cleaned this up, since `showUnMappedFields` is always equal `useNewFieldsApi`, feels like a leftover of our fields effort

Reply:
Yes, a leftover, long overdue for a cleanup 👍